downstream/modules/troubleshooting-aap/proc-troubleshoot-aap-packages.adoc

```diff
@@ -0,0 +1,6 @@
+[id="troubleshoot-aap-packages"]
+= Issue - Cannot locate certain packages that come bundled with the {PlatformNameShort} installer
+
+You cannot locate certain packages that come bundled with the {PlatformNameShort} installer, or you are seeing a "Repositories disabled by configuration" message.
+
+To resolve this issue you need to enable the repository by using the `subscription-manager` command in the command line. For more information about resolving this issue, see the _Troubleshooting_ section of link:{BaseURL}/red_hat_ansible_automation_platform/{PlatformVers}/html/red_hat_ansible_automation_platform_planning_guide/proc-attaching-subscriptions_planning[Attaching your {PlatformName} subscription] in the {PlatformName} Planning Guide.
```

Comment:
I would edit to say: To resolve this issue, ~~you need to~~ enable the repository by using the `subscription-manager` command in the command line...etc